src/Inciter/Partitioner.h

```diff
@@ -281,11 +273,13 @@ class Partitioner : public CBase_Partitioner< HostProxy,
         auto& inpoel = m_chinpoel[ c.first ];
         inpoel.insert( end(inpoel), begin(c.second), end(c.second) );
       }
-      Group::thisProxy[ frompe ].recv();
+      thisProxy[ frompe ].recv();
     }
 
     //! Acknowledge received node IDs
-    void recv() { if (--m_npe == 0) signal2host_distributed( m_host ); }
+    void recv() {
+      if (--m_npe == 0) contribute( m_cb.get< tag::distributed >() );
```

Comment:
I'm  a really big fan of adding some whitespace to the code, I feel it's pretty dense in places right now :)